explorer/src/context/main.tsx

```diff
@@ -1,17 +1,33 @@
+import { PaletteMode } from '@mui/material';
 import * as React from 'react';
+import { GatewayResponse, MixNodeResponse, ValidatorsResponse } from 'src/typeDefs/node-status-api-client';
+// import { MixNodeResponse, GatewayResponse } from 'src/typeDefs/node-status-api-client';
 import { Api } from '../api';
 
-export const MainContext = React.createContext({});
+type NodeApiResponse = {
+  data: MixNodeResponse | GatewayResponse | ValidatorsResponse | number | null
+  error: string | null
+}
```

Comment:
This isn't a good idea - you're undoing the power of TypeScript to check you by basically defining `any`